src/test/java/uk/gov/hmcts/reform/hmc/service/LinkedHearingGroupServiceTest.java

```diff
@@ -99,157 +87,10 @@ void shouldDeleteHearingGroupDetails() {
-
-            HearingEntity hearing = new HearingEntity();
-            hearing.setId(HEARING_ID1);
-            hearing.setStatus(HEARING_STATUS_UPDATE_REQUESTED);
-            hearing.setIsLinkedFlag(true);
-            hearing.setHearingResponses(List.of(
-                createHearingResponseEntityWithHearingDays(1, HEARING_RESPONSE_DATE_TIME,
-                                                           List.of(START_DATE_TIME_IN_THE_FUTURE,
-                                                                   START_DATE_TIME_IN_THE_PAST)
-                )));
-            LinkedGroupDetails groupDetails = createGroupDetailsEntity(HEARING_GROUP_ID, "ACTIVE");
-
-            given(linkedGroupDetailsRepository.findById(HEARING_GROUP_ID))
-                .willReturn(Optional.of(groupDetails));
-            given(hearingRepository.findByLinkedGroupId(HEARING_GROUP_ID))
-                .willReturn(List.of(hearing));
-
-            Exception exception = assertThrows(BadRequestException.class, () ->
-                service.deleteLinkedHearingGroup(HEARING_GROUP_ID));
-            assertEquals("008 Invalid state for unlinking hearing request " + HEARING_ID1,
-                         exception.getMessage());
-            verify(linkedGroupDetailsRepository, times(1)).findById(HEARING_GROUP_ID);
-            verify(hearingRepository, times(1)).findByLinkedGroupId(HEARING_GROUP_ID);
-            verify(linkedGroupDetailsRepository, never()).deleteHearingGroup(anyLong());
-        }
-
-        @Test
-        void shouldReturn400ErrorWhenHearingStatusIsInvalidForUnlinking() {
-
-            HearingEntity hearing = new HearingEntity();
-            hearing.setId(HEARING_ID1);
-            hearing.setStatus(FOR_DELETE_INVALID_HEARING_STATUS);
-            hearing.setIsLinkedFlag(true);
-            hearing.setHearingResponses(List.of(
-                createHearingResponseEntityWithHearingDays(1, HEARING_RESPONSE_DATE_TIME,
-                                                           List.of(START_DATE_TIME_IN_THE_FUTURE)
-                )));
-            LinkedGroupDetails groupDetails = createGroupDetailsEntity(HEARING_GROUP_ID, "ACTIVE");
-
-            given(linkedGroupDetailsRepository.findById(HEARING_GROUP_ID))
-                .willReturn(Optional.of(groupDetails));
-            given(hearingRepository.findByLinkedGroupId(HEARING_GROUP_ID))
-                .willReturn(List.of(hearing));
-
-            Exception exception = assertThrows(BadRequestException.class, () ->
-                service.deleteLinkedHearingGroup(HEARING_GROUP_ID));
-            assertEquals("008 Invalid state for unlinking hearing request " + HEARING_ID1,
-                         exception.getMessage());
-            verify(linkedGroupDetailsRepository, times(1)).findById(HEARING_GROUP_ID);
-            verify(hearingRepository, times(1)).findByLinkedGroupId(HEARING_GROUP_ID);
-            verify(linkedGroupDetailsRepository, never()).deleteHearingGroup(anyLong());
```

Comment:
this is the delete endpoint- why have we deleted these tests?

Reply:
Retrieved